### PR TITLE
Issue #240

### LINKED ISSUE #191: Review App Custom Tier Purchase
**Relays Values**
Please check if the values on the order summary are correct according to the Economics Formula values.
The values reference:
 https://www.notion.so/pocketnetwork/Purchase-Final-Numbers-9b4a110f8bcb4f63918f312c474713cb

**The order Summary** 
On the order Summary contains a "Balance" value which we need to subtract from the total cost of the purchasd amount of relays

Once completed the Purchase...

**Purchase Invoice**
There is an invoice generated on which some values should be pulled from the purchase information. Please, contact me for further details

**Notification**
There here is an alert generated which should remain over the app detail screen until the Network confirms the app Is staked. 

** Email Notification**
Once the app is staked an email (is already created on SendGrid) should be sent to the user.

**Fix APP DETAIL values**

Staked tokens and Balance should be expressed on POKT, not UPOKT

Max Relays Per Day should represent the just purchased  amount of relays

**NETWORKS**
This table should be showing the information about the stacked chains

### PR DESCRIPTION
**Pull request by pabelnl**
Closes #191 , #229 and #240 
- Remove app is only available for custom tier now.
- Fixed staked app chain list information not being properly retrieved.
- Multiple fixes and changes.

### CODE REVIEW

File: backend/src/services/ApplicationService.js [hunk 1]
```diff
@@ -675,9 +677,13 @@ export default class ApplicationService extends BasePocketService {
...
     const applicationDB = await this.persistenceService.getEntityByFilter(APPLICATION_COLLECTION_NAME, filter);
 
-    await this.persistenceService.deleteEntities(APPLICATION_COLLECTION_NAME, filter);
+    if (applicationDB.freeTier === false) {
+      await this.persistenceService.deleteEntities(APPLICATION_COLLECTION_NAME, filter);
 
-    return PocketApplication.createPocketApplication(applicationDB);
+      return PocketApplication.createPocketApplication(applicationDB);
+    } else {
+      throw new DashboardError("Free tier accounts can't be deleted.");
```
Comment (nymd) [inline backend/src/services/ApplicationService.js]: Free tier apps can't be deleted.

File: frontend/src/views/Apps/SelectRelays/SelectRelays.js [hunk 2]
```diff
@@ -263,39 +263,42 @@ class SelectRelays extends Component {
...
               className="max-alert"
               variant="primary"
-              title={<h4 className="alert-max">*More relays?</h4>}
+              title={<h4 className="alert-max">About RPD, Relays per day:</h4>}
             >
               <p className="alert-max">
-                If your app requires more than {formatNumbers(maxRelays)} Relays
-                per Day please <a href="mailto:dashboard@pokt.network">contact us</a> directly to find a
-                solution specially designed for your app.
+                Each RPD purchased on the Pocket Dashboard has a representation as a POKT token on the Pocket Network. Approx. 40 RPD=  1POKT.
```
Comment (nymd) [inline frontend/src/views/Apps/SelectRelays/SelectRelays.js]: Change:
"Approx. 40 RPD=  1POKT."
to
"Approx. 40 RPD = 1 POKT."

**Review by luyzdeleon — DISMISSED** [review]

File: frontend/src/views/Apps/TierSelection/FreeTierModal.js [hunk 3]
```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
...
-          <p>
-            As your app grows and scales you will be able to unstake and
-            transition to the Custom Tier to get the exact amount of relay
-            throughput you need.
-          </p>
-          <p>
-            If you still have questions please take a look at our{" "}
-            <a href="/todo">FAQ.</a>
+            Once your app scales up you can unstake and transition to the Stake and Scale plan to get the exact amount of
+            throughput you need. If you still have questions take a loot at our <a style={{fontWeight: "300", textDecoration: "none"}} href="https://dashboard.docs.pokt.network/docs/faq">FAQ</a>.
```
Comment (nymd) [inline frontend/src/views/Apps/TierSelection/FreeTierModal.js]: "take a look"

File: frontend/src/views/Apps/TierSelection/FreeTierModal.js [hunk 4]
```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
       >
         <Modal.Header closeButton></Modal.Header>
         <Modal.Body>
-          <h4>How the free tier works?</h4>
+        <h4>HOW IT WORKS? - LAUNCH OFFERING</h4>
           <p>
-            The free tier is a limited time infrastructure plan for any app to
-            receive free API throughput maxed at 1 Million Relays per Day.
+            In the Mainnet Launch Offering, Pocket Network Inc stakes on behalf of the customer and manages
+            the staked POKT as well as the AAT(Application Authentication Token). This will allow any app to access
```
Comment (nymd) [inline frontend/src/views/Apps/TierSelection/FreeTierModal.js]: as well as the AAT (Application Authentication Token).

File: frontend/src/views/Apps/TierSelection/FreeTierModal.js [hunk 5]
```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
...
-            In the free tier, Pocket Network Inc stakes on behalf of the user
-            and manages both the staked POKT and Application Authentication
-            Token (AAT). PNI reserves the right to revoke relay throughput at
-            any time for violating the Terms and Conditions. PNI is not
-            responsible for damage resulting from managing AATs or POKT.
+            In the Mainnet Launch Offering, Pocket Network Inc stakes on behalf of the app and manages the staked POKT and Application
+            Authentication Token (AAT). PNI reserves the right to revoke throughput at any time for violation of the <Link style={{fontWeight: "300", textDecoration: "none"}} to={ROUTE_PATHS.termsOfService}>Terms of Use</Link>.
+          </p>
+          <ul className="violation-list">
+            <li>Cases that may result in revocation of AAT include.</li>
```
Comment (nymd) [inline frontend/src/views/Apps/TierSelection/FreeTierModal.js]: This should be above the list, not a list item itself.

Comment (nymd) [reply frontend/src/views/Apps/TierSelection/FreeTierModal.js]: revocation of AAT include:
(colon, not period)

File: frontend/src/views/Apps/TierSelection/FreeTierModal.js [hunk 6]
```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
...
+            <li>Usage falls below 50,000 daily relays for over 3 months.</li>
+            <li>User is deemed to be using a fraudulent application or attacking the network.</li>
+            <li>User is exploiting the offering to run multiple free accounts.</li>
+            <li>After 1 Year of service, PNI will revisit continuning this service for all.</li>
```
Comment (nymd) [inline frontend/src/views/Apps/TierSelection/FreeTierModal.js]: This should be moved below the list into the next paragraph as it is not the same as the other list items.

"After one year of service,..."

File: frontend/src/views/Apps/TierSelection/FreeTierModal.js [hunk 7]
```diff
@@ -17,42 +21,38 @@ class FreeTierModal extends Component {
...
+          </ul>
+          <p>
+            PNI is not responsible for damage resulting from managing AATs or POKT.
           </p>
           <p>
-            If you need additional API bandwidth, you will always have the
-            option to upgrade to a paid tier that comes with additional control
-            over your AAT and POKT.
+            If you need additional bandwidth, you will always have the option to upgrade to a paid tier with additional control over your AAT and POKT.
+             It is recommended that you prepare for a surge by leveraging both this free and paid tiers in the event in order to not lose out on growth.
```
Comment (nymd) [inline frontend/src/views/Apps/TierSelection/FreeTierModal.js]: "It is recommended that you prepare for a surge by leveraging both this free and paid tiers in the event in order to not lose out on growth."
to
"If you are expecting rapid growth of your application, we suggest preparing for the surge by setting up the Stake and Scale paid tier so you don't experience downtime due to exhausting your available relays."

**Review by nymd — CHANGES_REQUESTED** [review]

**Review by nymd — APPROVED** [review]

**Pull request merged**